algorithms/unit_tests/CMakeLists.txt

```diff
@@ -56,6 +56,11 @@ IF(Kokkos_ENABLE_PTHREAD)
   )
 ENDIF()
 
+IF(Kokkos_ENABLE_HIP)
```

Comment:
Stick a comment in here something like: //KOKKOS_IMPL_HIP_WORKAROUND

Reply:
No there is a FIXME in the line below.  This is just temporary

Reply:
Yeah than at least do FIXME HIP so we can distinguish this from other FIXMEs